docs/0001-knowledge-graph-and-telemetry-specification/workload-tradeoff-service-api.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
is it a service stub or mock ?

Reply:
this is the example of tradeoff service response with the purpose to provide the mapping between the sources of knowledge graph to tradeoff service outputs. it is approximate and neither service stub nor mock.